`broker` crashes on `1.6.x` branch due to missing dependency

**Describe the bug**
On kapua relelase branch `1.6.x` the broker is crashing with the following logs
```
2023-11-14 16:15:55 15:15:55.553 [main] INFO  o.a.x.s.c.v.XBeanXmlBeanDefinitionReader - Loading XML bean definitions from class path resource [camel.xml]
2023-11-14 16:15:56 15:15:56.301 [main] ERROR o.a.a.xbean.XBeanBrokerFactory - Failed to load: class path resource [activemq.xml], reason: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56 org.springframework.beans.factory.BeanDefinitionStoreException: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:414)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.importBeanDefinitionResource(XBeanBeanDefinitionDocumentReader.java:143)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseDefaultElement(XBeanBeanDefinitionDocumentReader.java:109)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseBeanDefinitions(XBeanBeanDefinitionDocumentReader.java:95)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.doRegisterBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:142)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.registerBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:94)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.registerBeanDefinitions(XmlBeanDefinitionReader.java:508)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanXmlBeanDefinitionReader.registerBeanDefinitions(XBeanXmlBeanDefinitionReader.java:79)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:392)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.loadBeanDefinitions(ResourceXmlApplicationContext.java:116)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.loadBeanDefinitions(ResourceXmlApplicationContext.java:104)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractRefreshableApplicationContext.refreshBeanFactory(AbstractRefreshableApplicationContext.java:126)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractApplicationContext.obtainFreshBeanFactory(AbstractApplicationContext.java:614)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:514)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.<init>(ResourceXmlApplicationContext.java:64)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.<init>(ResourceXmlApplicationContext.java:52)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory$1.<init>(XBeanBrokerFactory.java:104)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory.createApplicationContext(XBeanBrokerFactory.java:104)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory.createBroker(XBeanBrokerFactory.java:67)
2023-11-14 16:15:56     at org.apache.activemq.broker.BrokerFactory.createBroker(BrokerFactory.java:71)
2023-11-14 16:15:56     at org.apache.activemq.broker.BrokerFactory.createBroker(BrokerFactory.java:54)
2023-11-14 16:15:56     at org.apache.activemq.console.command.StartCommand.runTask(StartCommand.java:87)
2023-11-14 16:15:56     at org.apache.activemq.console.command.AbstractCommand.execute(AbstractCommand.java:63)
2023-11-14 16:15:56     at org.apache.activemq.console.command.ShellCommand.runTask(ShellCommand.java:154)
2023-11-14 16:15:56     at org.apache.activemq.console.command.AbstractCommand.execute(AbstractCommand.java:63)
2023-11-14 16:15:56     at org.apache.activemq.console.command.ShellCommand.main(ShellCommand.java:104)
2023-11-14 16:15:56     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-11-14 16:15:56     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-11-14 16:15:56     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-11-14 16:15:56     at java.lang.reflect.Method.invoke(Method.java:498)
2023-11-14 16:15:56     at org.apache.activemq.console.Main.runTaskClass(Main.java:262)
2023-11-14 16:15:56     at org.apache.activemq.console.Main.main(Main.java:115)
2023-11-14 16:15:56 Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at java.lang.Class.getDeclaredMethods0(Native Method)
2023-11-14 16:15:56     at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
2023-11-14 16:15:56     at java.lang.Class.privateGetPublicMethods(Class.java:2902)
2023-11-14 16:15:56     at java.lang.Class.getMethods(Class.java:1615)
2023-11-14 16:15:56     at java.beans.Introspector.getPublicDeclaredMethods(Introspector.java:1336)
2023-11-14 16:15:56     at java.beans.Introspector.getTargetMethodInfo(Introspector.java:1197)
2023-11-14 16:15:56     at java.beans.Introspector.getBeanInfo(Introspector.java:426)
2023-11-14 16:15:56     at java.beans.Introspector.getBeanInfo(Introspector.java:173)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanQNameHelper.getBeanInfo(XBeanQNameHelper.java:78)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanQNameHelper.coerceNamespaceAwarePropertyValues(XBeanQNameHelper.java:54)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanBeanDefinitionParserDelegate.parseBeanDefinitionElement(XBeanBeanDefinitionParserDelegate.java:59)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.BeanDefinitionParserDelegate.parseBeanDefinitionElement(BeanDefinitionParserDelegate.java:459)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.BeanDefinitionParserDelegate.parseBeanDefinitionElement(BeanDefinitionParserDelegate.java:428)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.processBeanDefinition(XBeanBeanDefinitionDocumentReader.java:188)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseDefaultElement(XBeanBeanDefinitionDocumentReader.java:115)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseBeanDefinitions(XBeanBeanDefinitionDocumentReader.java:95)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.doRegisterBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:142)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.registerBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:94)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.registerBeanDefinitions(XmlBeanDefinitionReader.java:508)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanXmlBeanDefinitionReader.registerBeanDefinitions(XBeanXmlBeanDefinitionReader.java:79)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:392)
2023-11-14 16:15:56     ... 35 common frames omitted
2023-11-14 16:15:56 Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang.SerializationException
2023-11-14 16:15:56     at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
2023-11-14 16:15:56     at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
2023-11-14 16:15:56     at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
2023-11-14 16:15:56     ... 56 common frames omitted
2023-11-14 16:15:56 ERROR: java.lang.RuntimeException: Failed to execute start task. Reason: org.springframework.beans.factory.BeanDefinitionStoreException: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56 java.lang.RuntimeException: Failed to execute start task. Reason: org.springframework.beans.factory.BeanDefinitionStoreException: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at org.apache.activemq.console.command.StartCommand.runTask(StartCommand.java:91)
2023-11-14 16:15:56     at org.apache.activemq.console.command.AbstractCommand.execute(AbstractCommand.java:63)
2023-11-14 16:15:56     at org.apache.activemq.console.command.ShellCommand.runTask(ShellCommand.java:154)
2023-11-14 16:15:56     at org.apache.activemq.console.command.AbstractCommand.execute(AbstractCommand.java:63)
2023-11-14 16:15:56     at org.apache.activemq.console.command.ShellCommand.main(ShellCommand.java:104)
2023-11-14 16:15:56     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-11-14 16:15:56     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-11-14 16:15:56     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-11-14 16:15:56     at java.lang.reflect.Method.invoke(Method.java:498)
2023-11-14 16:15:56     at org.apache.activemq.console.Main.runTaskClass(Main.java:262)
2023-11-14 16:15:56     at org.apache.activemq.console.Main.main(Main.java:115)
2023-11-14 16:15:56 Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:414)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.importBeanDefinitionResource(XBeanBeanDefinitionDocumentReader.java:143)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseDefaultElement(XBeanBeanDefinitionDocumentReader.java:109)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseBeanDefinitions(XBeanBeanDefinitionDocumentReader.java:95)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.doRegisterBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:142)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.registerBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:94)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.registerBeanDefinitions(XmlBeanDefinitionReader.java:508)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanXmlBeanDefinitionReader.registerBeanDefinitions(XBeanXmlBeanDefinitionReader.java:79)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:392)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.loadBeanDefinitions(ResourceXmlApplicationContext.java:116)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.loadBeanDefinitions(ResourceXmlApplicationContext.java:104)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractRefreshableApplicationContext.refreshBeanFactory(AbstractRefreshableApplicationContext.java:126)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractApplicationContext.obtainFreshBeanFactory(AbstractApplicationContext.java:614)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:514)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.<init>(ResourceXmlApplicationContext.java:64)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.<init>(ResourceXmlApplicationContext.java:52)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory$1.<init>(XBeanBrokerFactory.java:104)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory.createApplicationContext(XBeanBrokerFactory.java:104)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory.createBroker(XBeanBrokerFactory.java:67)
2023-11-14 16:15:56     at org.apache.activemq.broker.BrokerFactory.createBroker(BrokerFactory.java:71)
2023-11-14 16:15:56     at org.apache.activemq.broker.BrokerFactory.createBroker(BrokerFactory.java:54)
2023-11-14 16:15:56     at org.apache.activemq.console.command.StartCommand.runTask(StartCommand.java:87)
2023-11-14 16:15:56     ... 10 more
2023-11-14 16:15:56 Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at java.lang.Class.getDeclaredMethods0(Native Method)
2023-11-14 16:15:56     at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
2023-11-14 16:15:56     at java.lang.Class.privateGetPublicMethods(Class.java:2902)
2023-11-14 16:15:56     at java.lang.Class.getMethods(Class.java:1615)
2023-11-14 16:15:56     at java.beans.Introspector.getPublicDeclaredMethods(Introspector.java:1336)
2023-11-14 16:15:56     at java.beans.Introspector.getTargetMethodInfo(Introspector.java:1197)
2023-11-14 16:15:56     at java.beans.Introspector.getBeanInfo(Introspector.java:426)
2023-11-14 16:15:56     at java.beans.Introspector.getBeanInfo(Introspector.java:173)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanQNameHelper.getBeanInfo(XBeanQNameHelper.java:78)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanQNameHelper.coerceNamespaceAwarePropertyValues(XBeanQNameHelper.java:54)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanBeanDefinitionParserDelegate.parseBeanDefinitionElement(XBeanBeanDefinitionParserDelegate.java:59)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.BeanDefinitionParserDelegate.parseBeanDefinitionElement(BeanDefinitionParserDelegate.java:459)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.BeanDefinitionParserDelegate.parseBeanDefinitionElement(BeanDefinitionParserDelegate.java:428)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.processBeanDefinition(XBeanBeanDefinitionDocumentReader.java:188)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseDefaultElement(XBeanBeanDefinitionDocumentReader.java:115)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseBeanDefinitions(XBeanBeanDefinitionDocumentReader.java:95)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.doRegisterBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:142)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.registerBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:94)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.registerBeanDefinitions(XmlBeanDefinitionReader.java:508)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanXmlBeanDefinitionReader.registerBeanDefinitions(XBeanXmlBeanDefinitionReader.java:79)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:392)
2023-11-14 16:15:56     ... 35 more
2023-11-14 16:15:56 Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang.SerializationException
2023-11-14 16:15:56     at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
2023-11-14 16:15:56     at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
2023-11-14 16:15:56     at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
2023-11-14 16:15:56     ... 56 more
2023-11-14 16:15:56 ERROR: org.springframework.beans.factory.BeanDefinitionStoreException: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56 org.springframework.beans.factory.BeanDefinitionStoreException: Unexpected exception parsing XML document from class path resource [camel.xml]; nested exception is java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:414)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.importBeanDefinitionResource(XBeanBeanDefinitionDocumentReader.java:143)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseDefaultElement(XBeanBeanDefinitionDocumentReader.java:109)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseBeanDefinitions(XBeanBeanDefinitionDocumentReader.java:95)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.doRegisterBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:142)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.registerBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:94)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.registerBeanDefinitions(XmlBeanDefinitionReader.java:508)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanXmlBeanDefinitionReader.registerBeanDefinitions(XBeanXmlBeanDefinitionReader.java:79)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:392)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:336)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.loadBeanDefinitions(ResourceXmlApplicationContext.java:116)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.loadBeanDefinitions(ResourceXmlApplicationContext.java:104)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractRefreshableApplicationContext.refreshBeanFactory(AbstractRefreshableApplicationContext.java:126)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractApplicationContext.obtainFreshBeanFactory(AbstractApplicationContext.java:614)
2023-11-14 16:15:56     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:514)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.<init>(ResourceXmlApplicationContext.java:64)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.ResourceXmlApplicationContext.<init>(ResourceXmlApplicationContext.java:52)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory$1.<init>(XBeanBrokerFactory.java:104)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory.createApplicationContext(XBeanBrokerFactory.java:104)
2023-11-14 16:15:56     at org.apache.activemq.xbean.XBeanBrokerFactory.createBroker(XBeanBrokerFactory.java:67)
2023-11-14 16:15:56     at org.apache.activemq.broker.BrokerFactory.createBroker(BrokerFactory.java:71)
2023-11-14 16:15:56     at org.apache.activemq.broker.BrokerFactory.createBroker(BrokerFactory.java:54)
2023-11-14 16:15:56     at org.apache.activemq.console.command.StartCommand.runTask(StartCommand.java:87)
2023-11-14 16:15:56     at org.apache.activemq.console.command.AbstractCommand.execute(AbstractCommand.java:63)
2023-11-14 16:15:56     at org.apache.activemq.console.command.ShellCommand.runTask(ShellCommand.java:154)
2023-11-14 16:15:56     at org.apache.activemq.console.command.AbstractCommand.execute(AbstractCommand.java:63)
2023-11-14 16:15:56     at org.apache.activemq.console.command.ShellCommand.main(ShellCommand.java:104)
2023-11-14 16:15:56     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-11-14 16:15:56     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-11-14 16:15:56     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-11-14 16:15:56     at java.lang.reflect.Method.invoke(Method.java:498)
2023-11-14 16:15:56     at org.apache.activemq.console.Main.runTaskClass(Main.java:262)
2023-11-14 16:15:56     at org.apache.activemq.console.Main.main(Main.java:115)
2023-11-14 16:15:56 Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/SerializationException
2023-11-14 16:15:56     at java.lang.Class.getDeclaredMethods0(Native Method)
2023-11-14 16:15:56     at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
2023-11-14 16:15:56     at java.lang.Class.privateGetPublicMethods(Class.java:2902)
2023-11-14 16:15:56     at java.lang.Class.getMethods(Class.java:1615)
2023-11-14 16:15:56     at java.beans.Introspector.getPublicDeclaredMethods(Introspector.java:1336)
2023-11-14 16:15:56     at java.beans.Introspector.getTargetMethodInfo(Introspector.java:1197)
2023-11-14 16:15:56     at java.beans.Introspector.getBeanInfo(Introspector.java:426)
2023-11-14 16:15:56     at java.beans.Introspector.getBeanInfo(Introspector.java:173)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanQNameHelper.getBeanInfo(XBeanQNameHelper.java:78)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanQNameHelper.coerceNamespaceAwarePropertyValues(XBeanQNameHelper.java:54)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2c.XBeanBeanDefinitionParserDelegate.parseBeanDefinitionElement(XBeanBeanDefinitionParserDelegate.java:59)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.BeanDefinitionParserDelegate.parseBeanDefinitionElement(BeanDefinitionParserDelegate.java:459)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.BeanDefinitionParserDelegate.parseBeanDefinitionElement(BeanDefinitionParserDelegate.java:428)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.processBeanDefinition(XBeanBeanDefinitionDocumentReader.java:188)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseDefaultElement(XBeanBeanDefinitionDocumentReader.java:115)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanBeanDefinitionDocumentReader.parseBeanDefinitions(XBeanBeanDefinitionDocumentReader.java:95)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.doRegisterBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:142)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.DefaultBeanDefinitionDocumentReader.registerBeanDefinitions(DefaultBeanDefinitionDocumentReader.java:94)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.registerBeanDefinitions(XmlBeanDefinitionReader.java:508)
2023-11-14 16:15:56     at org.apache.xbean.spring.context.v2.XBeanXmlBeanDefinitionReader.registerBeanDefinitions(XBeanXmlBeanDefinitionReader.java:79)
2023-11-14 16:15:56     at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.doLoadBeanDefinitions(XmlBeanDefinitionReader.java:392)
2023-11-14 16:15:56     ... 35 more
2023-11-14 16:15:56 Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang.SerializationException
2023-11-14 16:15:56     at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
2023-11-14 16:15:56     at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
2023-11-14 16:15:56     at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
2023-11-14 16:15:56     ... 56 more
```
This seems to be related to the missing dependency `commons-lang:commons-lang`.

**To Reproduce**
Steps to reproduce the behavior:

1. Run kapua on `1.6.x branch`
2. check `broker` logs

**Expected behavior**
The broker should work fine

**Version of Kapua**
1.6.x

**Type of deployment**
[ X] Docker
[ ] Openshift (in its variants)
[ ] Others

--- a/assembly/broker/descriptors/kapua-broker.xml
+++ b/assembly/broker/descriptors/kapua-broker.xml
@@ -171,6 +171,7 @@
                 <include>commons-collections:commons-collections</include>
                 <include>commons-configuration:commons-configuration</include>
                 <include>commons-io:commons-io</include>
+                <include>commons-lang:commons-lang</include>
 
                 <include>io.dropwizard.metrics:metrics-core</include>
                 <include>io.netty:netty</include>

--- a/assembly/broker/pom.xml
+++ b/assembly/broker/pom.xml
@@ -391,6 +391,11 @@
             <artifactId>javax.annotation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
+        </dependency>
+        <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>

